Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ruby:2.3-onbuild
+
+RUN \
+    apt-get update \
+    && apt-get install -y --no-install-recommends nodejs \
```

Comment:
you might want to use `apt-get clean` here to remove archives from the apt cache:

```
clean
    clean clears out the local repository of retrieved package files. It removes everything but the lock file from /var/cache/apt/archives/ and /var/cache/apt/archives/partial/
````